.github/workflows/ci.yaml

```diff
@@ -452,6 +452,11 @@ jobs:
           . venv/bin/activate
           python -m pip install -U pip setuptools wheel
           pip install -U -r requirements_test_min.txt
+      - name: Install pytest
+        run: |
+          python -m venv venv
+          . venv/bin/activate
+          python -m pip install --force-reinstall pytest
```

Comment:
?

Reply:
This is a failed attempt to fix the CI. The `pytest` `pypy` job runs with Python 3.8 even though we download and set up a Python 3.7 version. I thought there might be a previous `pytest` installation in the `venv` that uses 3.8 but this didn't seem to fix it.
Haven't looked into it further, but I think we should be able to use 3.7 if we can `pytest` to actually use it as well.

Reply:
Just a heads up, I'm seeing failing tests in this run as well. I think the `venv` might be messed up now 😓

Reply:
Try to bump the cache env variable

Reply:
Ah, now I understand what that's used for. Clever trick 😉